SalesforceSDK/Salesforce.SDK.Hybrid.SmartStore/Source/Store/SmartStore.cs

```diff
@@ -165,6 +166,16 @@ public object Upsert(string soupName, string soupElt, string externalIdPath, boo
             return NativeSmartStore.Upsert(soupName, JObject.Parse(soupElt), externalIdPath, handleTx);
         }
 
+        public bool BeginDatabaseTransaction()
```

Comment:
discussed offline with @kchitalia, yes we need this to perfom bulk operations efficiently